### ISSUE TITLE
Assign css template literals to an object's key

### ISSUE THREAD

**Issue by berbaquero** [callstack/linaria#138]
**Do you want to request a *feature* or report a *bug*?**
Request feature.

**What is the current behavior?**
Linaria only allows to assign `css` template literals to a variable: 

```js
// works nicely
const style = css`
  color: white;
  display: inline-block;
`;
```

The babel plugin fails when assigning to an object's key:

```js
// fails
const style = {
  wrapper: css`
    color: black;
  `,
};
```

**If the current behavior is a bug, please provide the steps to reproduce and a minimal repository on GitHub that we can `yarn install` and `yarn test`.**
...

**What is the expected behavior?**

An alternative convention is to declare all the styles of a component in a single object:

```jsx
const styles = {
  root: css`
    color: white;
  `,
  header: css`
    padding: 3em;
  `,
}

<div className={styles.root}>
  <header className={styles.header}>
```

**Please provide your exact Babel configuration and mention your Linaria, Node, Yarn/npm version and operating system.**
...

**Comment by satya164** [comment]
We're working on supporting this :)

**Comment by zamotany** [comment]
The bug should be fixed by #134
